### DISCUSSION TITLE
빌드잇 오브젝트 맵핑이 오류가 있는 것 같아요.

### DISCUSSION THREAD

**Discussion by mahoworld** [naverz/zepeto-studio-kor#449] [Issue Report (Legacy)]
<a href='https://ifh.cc/v-9Jdal2' target='_blank'><img src='https://ifh.cc/g/9Jdal2.jpg' border='0'></a>

안녕하세요.

빌드잇 맵에서 이용할 수 있는 오브젝트 중 Driving 테마에서

잔디를 포함한 오브젝트가 유니티에 로드 했을 때 메쉬가 이상해요! 

혹시 해결 방법이 있을까요?

**Comment by crash-k** [comment]
안녕하세요.
혹시 해당 이슈를 좀 더 자세하게 확인해 볼 수 있도록 맵 코드를 알려 주실 수 있을까요?

**Comment by mahoworld** [comment]
안녕하세요.
맵 코드는 0FDBA1D4 입니다.
처음에는 일반적인 맵을 불러왔다가 해당 문제를 확인하고 로드에 오류가 있는 것인지 아니면 오브젝트 자체에 오류가 있는 것인지 확인하기 위해
테스트용으로 Driving 테마의 오브젝트를 무작위로 배치한 맵입니다.
 
빌드잇 프로그램 상에서는 문제가 발생하지 않습니다만,
유니티에서 빌드잇 모델링을 불러올 때 잔디가 있는 Driving 테마의 특정 오브젝트 맵핑이 오류가 발생하고 있습니다.
혹시 오브젝트 상에서의 문제가 아니라면 제가 스스로 해결할 수 있는 방법도 있을까요 ...?
바쁘시겠지만 꼭 좀 알려주셨으면 합니다...
 
 
-----Original Message-----

**Reply by crash-k** [reply]
안녕하세요. @mahoworld 님

우선 사용에 불편을 드려 죄송합니다. 아직 프리뷰로 운영되는 패키지이다 보니 미흡한 점이 있었습니다.
제공하신 정보를 바탕으로 문제를 인지하고 수정해서 0.0.3 버전으로 긴급 배포하였습니다.
번거로우시겠지만 zepeto.buildit.loader를 0.0.3으로 업데이트하여 다시 필요 맵을 로드해 주시면 수정이 될 것 같습니다.

감사합니다.
